hw2/05-form.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>05 - Form</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
+    <script defer src="05-form.js"></script>
+  </head>
+  <body class="bg-dark">
+    <main>
+      <form class="mx-auto mt-5 w-50 border p-3" name="form-data">
+        <h1 class="text-center mt-2">Exercise 05 - Form</h1>
+        <div class="mb-3">
+          <label for="name" class="form-label h6">Full Name:</label>
+          <input id="name" type="text" class="form-control" name="name" />
+        </div>
+
+        <div class="mb-3">
+          <label for="email" class="form-label h6">Email:</label>
+          <input id="email" type="email" class="form-control" name="email" />
+        </div>
+
+        <div class="mb-3">
+          <label for="registrationStatus" class="form-label h6"
+            >Registration Status</label
+          >
+          <select
+            id="registrationStatus"
+            class="form-select h6"
+            name="regStatus"
+          >
+            <option value="Choose an Option">Choose an Option</option>
+            <option value="Registered">Registered</option>
+            <option value="Cancelled">Cancelled</option>
+            <option value="Wait List">Wait List</option>
+          </select>
+        </div>
+
+        <div class="mb-3">
```

Comment:
WAVE Audit Error: I ran the WAVE audit and it looks like there is a missing fieldset for your checkboxes I think adding that will make the alert go away. Besides that great job on the form and you have a very clean implementation.